Vungle/VungleRouter.m

```diff
@@ -538,6 +538,28 @@ - (void)cleanupFullScreenDelegate:(id<VungleRouterDelegate>)delegate
     }
 }
 
+- (void)removeDelegatesIfContainsPlacement:(NSString *)placementID
+                                dictionary:(NSMutableDictionary *)dictionary
+{
```

Comment:
Might we need to do thread synchronization here? Thanks a lot. :)

Reply:
Updated.